Academia-750/academia750-frontend-vue
# DBT-926 Feat/separate student material

## Context


Refactoring  by removing lesson from the reducer and creating one component


## Test

- Login as a student
- click on materials and recordings you should see materials or recordings without filter
- when you click to a class and see it's recordings or materials then it filters





## Platform



## Proof of Concept
<img width="1045" alt="image" src="https://github.com/Academia-750/academia750-frontend-vue/assets/79788219/a7959d03-243c-416c-88e1-fa9f64b490bf">
